js/components/controlBar.js

```diff
@@ -317,6 +317,26 @@ var ControlBar = createReactClass({
     this.props.controller.toggleMoreOptionsScreen(this.moreOptionsItems);
   },
 
+  filterAudioControls: function() {
```

Comment:
As far as I see this method doesn't filter. It produces new config block based on the retrieved from server. I'd call it `getAudioControlsConfig`. Since it's not a simple getter would be great to have a comment describing the transformations doing.